jcoin.datacommons.io/portal/gitops.json

```diff
@@ -492,16 +492,20 @@
           {
             "title": "Constructs",
             "fields": [
-              "cmc_data_type",
+              "cmc_subject",
               "cmc_title"
             ]
```

Comment:
in QA we have:
```
            "fields": [
              "cmc_data_type",
              "cmc_subject",
              "cmc_title"
            ]
```

Reply:
It'll break in QA but not PROD because [gitops.json for PROD has cmc_subject](https://github.com/uc-cdis/cdis-manifest/blob/b4fe17d55a069da2f9f3b67db8df8e11d4dd602e/jcoin.datacommons.io/portal/gitops.json#L493). I'll change the QA to correct version